utils/gitlab_api.py

```diff
@@ -0,0 +1,97 @@
+import gitlab
+import urllib3
+import uuid
+from reconcile.config import get_config
+
+
+class GitLabApi(object):
+    def __init__(self, project_name, ssl_verify=False):
```

Comment:
I would change this to be:

`def __init__(self, server, token, project_id, ssl_verify=True):`

and create the gl api internally here:

```
self.gl = gitlab.Gitlab(server, private_token=token, ssl_verify=ssl_verify)
self.project = self.gl.projects.get(project_id)
```

Reply:
note that I would default to ssl_verify=True